polygon/bor/bor.go

```diff
@@ -385,7 +386,7 @@ func New(
 		execCtx:                context.Background(),
 		logger:                 logger,
 		closeCh:                make(chan struct{}),
-		bridgeReader:           bridgeReader,
+		BridgeReader:           bridgeReader,
```

Comment:
would suggest to keep the reader private as per https://github.com/erigontech/erigon/pull/11484/files#r1705561153